services/provider/proto/provider.proto

```diff
@@ -147,8 +149,10 @@ message FulfillStorageClassClaimResponse{
 // RevokeStorageClassClaimRequest holds the information required to delete the
 // StorageclassClaim CR on provider cluster.
 message RevokeStorageClassClaimRequest{
-    // storageclassClaim Identified on the provider cluster.
-    string storageClassClaimID = 1;
+    // name of the storageclassclaim on the consumer cluster.
+    string storageClassClaimName = 1;
```

Comment:
What happens on provider, when user deletes and recreates claim on consumer cluster with same name but different spec? Would UUID be better here?

We need a mechanism to cancel older request when we see new updates on claim or it gets re-created.

Reply:
Cancellation can lead to unwanted side effects. the recreation will not go through as the object already exists and it should wait till the deletion is completed.

Reply:
@umangachapagain we did that by design, it is not a side effect